xbmc/interfaces/builtins/GUIBuiltins.cpp

```diff
@@ -417,6 +471,10 @@ CBuiltins::CommandMap CGUIBuiltins::GetOperations() const
            {"action",                         {"Executes an action for the active window (same as in keymap)", 1, Action}},
            {"cancelalarm",                    {"Cancels an alarm", 1, CancelAlarm}},
            {"alarmclock",                     {"Prompt for a length of time and start an alarm clock", 2, AlarmClock}},
+           {"canceltimeout",                  {"Cancels the timer", 1, CancelTimer}},
+           {"timeout",                        {"Executes a built-in command after a specified number of milliseconds", 3, Timeout}},
```

Comment:
That doesnt change that it is misleading. ;)
"Timeout: a cancellation or cessation that automatically occurs when a predefined interval of time has passed without a certain event occurring."